react/class_components/class_based_components.md

```diff
@@ -275,15 +275,15 @@ For the purposes of this assignment, we take the class-based component that we b
 
 ### Knowledge check
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
 - [How do props get used in a class-based component?](#the-use-of-a-constructor-and-props)
 - [How does JSX get displayed?](#how-you-can-render-jsx)
 - [How do we deal with state in a class-based component?](#how-to-use-state-and-manage-context)
-- [How do you restore the context of `this` in a method?](#the-importance-of-bind)
```

Comment:
Nit: "this" being mentioned here may be important. Was it just removed to appease the linter?

Reply:
Yeah, the removal of 'this' from the link label was indeed to adhere to the markdown lint rule.

Reply:
@roopxx can you confirm whether that was caught by the linter or if you just manually updated it? Just wondering because "this" in backticks when referencing the `this` keyword shouldn't be getting flagged by the linter as that is a valid use of the word (referencing a keyword rather than just using the word "this").

Reply:
@thatblindgeye it was caught by the linter and gave me the TOP001 - descriptive-link-text error.  I know how important the keyword `this` is here but it was throwing lint error when I ran the markdown in CLI.

Reply:
@thatblindgeye and @KevinMulhern I've double-checked, and it seems that the original label text was actually fine and didn't trigger any lint errors. I apologize for the oversight. I must have missed something during the initial change. I've reverted the specific commit changes to ensure the correct labelling is in place. Thanks again for your patience.

Reply:
No worries! The update made to that custom rule that allows `this` in backticks was relatively recent I believe, so we had wondered if maybe your PR was being checked against a previous commit of the rule or something.